### PR TITLE
Mig 17.0 mail_unsibscribe

### PR DESCRIPTION
**Pull request by laszlokr**
@janikvonrotz following test instructions I could not verify if `mail_create_nosubscribe` option still works

### CODE REVIEW

**Comment by janikvonrotz** [comment]
@laszlokr At which point in the test instructions where not able to verify the statement?

**Comment by laszlokr** [comment]
@janikvonrotz basically on the last step:
- Create a new quotation and send it, the dialog will have no subscribers

The recipients of the quotation are still added back to the followers list. I also tested when only this option is enabled, and the other two are not, but I still could not see that there are zero followers after the quotation is sent.

File: mail_unsubscribe/models/mail_thread.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            rec.message_unsubscribe(message_partner_ids.ids)
+
+    @api.returns("mail.message", lambda value: value.id)
+    def message_post(self, **kwargs):
+        mail_post_autofollow = ast.literal_eval(
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("mail_unsubscribe.mail_post_autofollow", "False")
+        )
+        mail_create_nosubscribe = ast.literal_eval(
```
Comment (janikvonrotz) [inline mail_unsubscribe/models/mail_thread.py:36]: Log this value.

File: mail_unsubscribe/models/mail_thread.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("mail_unsubscribe.mail_create_nosubscribe", "False")
+        )
+
+        if self._name != "mail.channel":
+            self._unsubscribe_followers()
+            _self = self.with_context(
```
Comment (janikvonrotz) [inline mail_unsubscribe/models/mail_thread.py]: Waht is `_self`? Remove the underscore and check if it works.

**Review by janikvonrotz — APPROVED** [review]

**Pull request merged**